grapher/core/Grapher.tsx

```diff
@@ -832,12 +825,8 @@ export class Grapher
         return findClosestTime(this.times, this.endHandleTimeBound)
     }
 
-    @computed private get isDiscreteBarOrLineChartTransformedIntoDiscreteBar() {
-        return (
-            this.typeExceptWhenLineChartAndSingleTimeThenWillBeBarChart ===
-                ChartTypeName.DiscreteBar ||
-            (this.type === ChartTypeName.LineChart && this.isPlaying)
-        )
+    @computed private get onlySingleTimeSelectionPossible() {
+        return this.type === ChartTypeName.DiscreteBar || this.isOnMapTab
```

Comment:
Nice! Simpler logic and more readable

Reply:
I'm not 100% sure about the name, what do you think about it?

Reply:
It's definitely an improvement! It could mention `handle` maybe? But this is good enough